### ISSUE TITLE
MWE - Error on single dim targets

### ISSUE THREAD

**Issue by drozzy** [lorenzoh/DataLoaders.jl#16]
Here is the MWE for the error that I encountered in #14:

```
x = rand(10, 10000)
y = rand(10000)
dataloader = DataLoader((x, y ), BATCH)
```

In my dataset, the targets (y's) are just integers.
However, it seems data loader can't deal with that.

How would I resolve that?

Thank you!

**Comment by drozzy** [comment]
Also, it hangs after this error occurs. It would be nice if it terminated cleanly.

**Comment by drozzy** [comment]
Nevermind, this  doesn't break it.
